xbmc/playlists/SmartPlayList.cpp

```diff
@@ -75,6 +75,7 @@ static const translateField fields[] = {
   { "lastplayed",        FieldLastPlayed,              CDatabaseQueryRule::DATE_FIELD,     NULL,                                 false, 568 },
   { "inprogress",        FieldInProgress,              CDatabaseQueryRule::BOOLEAN_FIELD,  NULL,                                 false, 575 },
   { "rating",            FieldRating,                  CDatabaseQueryRule::NUMERIC_FIELD,  CSmartPlaylistRule::ValidateRating,   false, 563 },
+  { "userrating",        FieldUserRating,              CDatabaseQueryRule::NUMERIC_FIELD,  CSmartPlaylistRule::ValidateRating,   false, 38018 },
```

Comment:
Not sure if CSmartPlaylistRule::ValidateRating handles int fine. As it seems to be written for decimals.